.github/workflows/terraform-rds-irsa-test.yaml

```diff
@@ -0,0 +1,68 @@
+name: Terraform rds/s3 Test
+on:
+  # Run on PR configuration
+  push:
+    paths:
+      - deployments/rds-s3/terraform/**
+      - iaac/terraform/apps/**
+      - iaac/terraform/common/**
+      - iaac/terraform/utils/**
+    branches:
+      - main
+
+  # Run on schedule (every 3 hours)
```

Comment:
Is this too often? How long does the rds-s3 test usually take for full setup around 1.5 hrs ?

Reply:
It will just be for 1-2 days. I will disable the workflow after the release